pages/spec/controllers/refinery/pages_controller_spec.rb

```diff
@@ -3,20 +3,33 @@
 module Refinery
   describe PagesController, :type => :controller do
     before do
-      FactoryBot.create(:page, :link_url => "/")
-      FactoryBot.create(:page, :title => "test")
+      FactoryBot.create(:page, link_url:  "/")
+      FactoryBot.create(:page, title:  "testing")
+      FactoryBot.create(:page, link_url: "/items")
     end
 
     describe "#home" do
-      it "renders home template" do
-        get :home
-        expect(response).to render_template("home")
+      context "when page path set to default ('/') " do
+        render_views
+        it "renders home template" do
+          get :home
+          expect(response).to render_template("home")
+        end
+      end
+
+      context "when home_page_path set to another path" do
+        it "redirects to the specified path" do
+          allow(Refinery::Pages).to receive(:home_page_path).and_return('/items')
+          get :home
+          expect(response).to redirect_to('/items')
+        end
       end
     end
 
     describe "#show" do
+      render_views
```

Comment:
how come this needs `render_views` now?

Reply:
Because when I ran the tests it came back from this (and another similar test) saying the view was rendered with `[]`. (from memory).

Reply:
When I remove `render_views` from the file completely the tests still pass. 🤔

Reply:
I did the same (removed the 'render_views' and also got no failures. So out they go.